src/S3/audio.c

```diff
@@ -793,6 +793,18 @@ tS3_sound_tag S3StartSound(tS3_outlet* pOutlet, tS3_sound_id pSound) {
         gS3_last_error = eS3_error_bad_id;
         return 0;
     }
+#if defined(S3_FIX_BUGS)
```

Comment:
I think we could use `DETHRACE_FIX_BUGS` here - its probably unlikely someone would want to fix bugs in one area but not the other